### DISCUSSION TITLE
I am lost.

### DISCUSSION THREAD

**Discussion by alanecn** [Elskom/Els_kom_new#184] [Q&A]
Hello, I need some help.
I am just looking to extract Elsword's assets from its kom files for editing projects (not to mod or anything else like that, I just want them tasty pngs to plug into other projects) and since I am no coder, just a photoshop gremlin, I am completely lost.
I installed Git and Visual Studio 2019, and tried to follow the instructions on the readme, however, I have no idea how to "run" the project solution file to get what I want.

So I checked out the other discussion, and tried following that, but Git kept giving me errors relating to a 1.0.0 that isnt there??? Also you mentioned there being a koms file to drop koms into but there is no such thing.

**Comment by AraHaan** [comment]
the koms folder is created when you first start the program. As for the other thing I separated a lot of it’s dependencies into it’s own sdk and runtime however there is a current issue I face trying to work around it with loading it’s runtime as the default apphost identifies it as a missing 6.0.1 “.NET Desktop Runtime" which is not missing, it just erroneously expects all of the dependent runtimes to be the same exact version sadly for some reason.
